pages/getting-started/developers.md

```diff
@@ -80,24 +80,24 @@ node_modules/uswds/
 │   ├── js/
 └── src/
     ├── fonts/
-    ├── img/
-    ├── js/
-    └── stylesheets/
-```
+     ├── img/
+      ├── js/
+       └── stylesheets/
+    ```
 
```

Comment:
The extra spaces seems to be causing some styling issues. See screenshot below:

![screen shot 2016-12-07 at 4 42 55 pm](https://cloud.githubusercontent.com/assets/955558/20987891/5ae3837a-bc9c-11e6-964b-c77f19effcc0.png)